crates/sui-faucet/src/faucet/simple_faucet.rs

```diff
@@ -131,6 +135,7 @@ impl SimpleFaucet {
             request_producer: Mutex::new(sender),
             request_consumer: Mutex::new(receiver),
             batch_request_size: config.batch_request_size,
+            task_id_cache: TtlCache::new(480000).into(),
```

Comment:
Will configure this number later for number of max requests we want to serve * duration period we hold requests